Dockerfile

```diff
@@ -5,6 +5,9 @@ LABEL version="1.0" \
       description="Probot app which is a modified version of Settings Probot GitHub App" \
       maintainer="GitHub Professional Services <services@github.com>"
 
+RUN   mkdir /opt/safe-settings \
+  &&  chown -R node:node /opt/safe-settings
+
 USER node
 ## Set our working directory
 WORKDIR /opt/safe-settings
```

Comment:
Yes, this should be enough to create the /op/safe-settings dir. We don't need to explicitly create it like in the previous version.